services/frontend/src/utils/models/Project.test.js

```diff
@@ -117,19 +117,28 @@ describe("mergeWithSkills", () => {
     const skillB = new Skill({id: "b", name: "test2"});
     const skillC = new Skill({id: "c", name: "test3"});
 
-    const project1WithSkills = {...project1, skills: [skillA.name, skillB.name]};
-    const project2WithSkills = {...project2, skills: [skillB.name, skillC.name]};
+    /* We delete the createdAt and updatedAt then use toMatchObject instead of toEqual
```

Comment:
I don't get _why_ you only want to test against a subset of the properties. What's stopping you from just using `toEqual` with the full object?

Reply:
I realize now that I can use `toEqual` again so I have swapped back to it.

When I was using `new Skill()` inline it was creating the object at a different time than the skill object passed in so `createdAt` and `updatedAt` would always be different. now that I am using 
```
acc = [...acc, skillsById[skillId]];
```
toEqual is alright again.